sources/packages/web/src/views/student/StudentEditApplication.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <v-btn color="primary" class="float-right" @click="toggle"
+    >Application Options
+    <v-icon size="25"> mdi-arrow-down-bold-circle</v-icon></v-btn
+  >
+  <Menu class="mt-n15" ref="menu" :model="items" :popup="true" />
```

Comment:
Is there any other Vue component that we can use instead of this PrimeVue?

Reply:
@andrewsignori-aot I checked this before using primevue, vuetify3 alfa, does not support v-menu as of now